docs/source/ops.rst

```diff
@@ -93,3 +93,13 @@ State Space Model and GP Utilities
     :undoc-members:
     :show-inheritance:
     :member-order: bysource
+
+
+Numerical Methods Utilities
```

Comment:
a bit vague?

Reply:
Do you think "Numerical Methods for Interpolation" might be a better title? If someone implements ODE solvers in the future for stochasic ODEs, should that be under a separate heading like "Numerical Methods for Differential Equations"?

Reply:
Well you could first add smoke test exercising the new functions and methods with scalar and batched inputs. If you want to add further tests, you could add say a spline approximation of a non-polynomial function like `sin(x)` or `sin(x) exp(-x**2/200)` and `assert_close(spline_approx, truth)`.